hub-database.sh

```diff
@@ -48,14 +49,26 @@ hostnossl all             	all                   0.0.0.0/0                 rejec
 EOF
 fi
 
-## Certificate management
+dataPopulated=false
+
+if [ -d "$PGDATA" ] && [ -s "$PGDATA/$PG_VERSION" ];
+then
+	echo "Data directory not populated"
```

Comment:
I think the "not" is in the wrong branch.

Reply:
Corrected